api-common-java/src/test/java/com/google/api/core/ApiFuturesTest.java

```diff
@@ -38,12 +38,12 @@
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Executor;
 import java.util.concurrent.atomic.AtomicInteger;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 public class ApiFuturesTest {
```

Comment:
I believe the class can also be package-private: i.e. `class ApiFuturesTest`

Reply:
Ah okay - I'll update all of them to be package-private, assuming there's no reason for test classes to be public?

Reply:
I think they were required for Junit4, but Junit5 no longer requires them to be public (just not private).